From 32845d98ac7cfe3558a32fcc351d042549cb4a3e Mon Sep 17 00:00:00 2001
From: Adrien Ricciardi <aricciardi@baylibre.com>
Date: Mon, 18 Sep 2023 20:21:05 +0200
Subject: [PATCH 05/38] machines: cortex-a: Fixed an unused parameter warning
 in tpl_check_stack_footprint().

The 'proc_id' parameter is not used and triggers a compilation warning.

Signed-off-by: Adrien Ricciardi <aricciardi@baylibre.com>
---
 machines/cortex-a/tpl_machine_arm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/machines/cortex-a/tpl_machine_arm.c b/machines/cortex-a/tpl_machine_arm.c
index 91b1a0df..e3c6817a 100644
--- a/machines/cortex-a/tpl_machine_arm.c
+++ b/machines/cortex-a/tpl_machine_arm.c
@@ -210,6 +210,9 @@ FUNC(uint8, OS_CODE) tpl_check_stack_footprint (
     CONST(tpl_proc_id, OS_APPL_DATA) proc_id)
 {
   uint8 tmp;
+
+  (void) proc_id;
+
   /*to do*/
   tmp=0;
   return tmp;
-- 
2.34.1

